assets/javascripts/discourse/initializers/discourse-calendar.js

```diff
@@ -295,11 +300,21 @@ function initializeDiscourseCalendar(api) {
         },
       },
       header: {
-        left: "prev,next today",
+        left: "prev,next today timezoneOffset",
         center: "title",
         right: "month,basicWeek,listNextYear",
       },
       eventOrder: ["start", _orderByTz, "-duration", "allDay", "title"],
+      customButtons:
+        siteSettings.enable_timezone_offset_for_calendar_events && {
+          timezoneOffset: {
+            text: "Timezone Offset",
```

Comment:
Can we use the `globe` icon instead of a text label? It would take less space in the screen. You can try with either `iconNode("globe")` or `iconHTML("globel")` helpers.

Reply:
The default icon support with the FullCalendar library is very limited. It doesn't allow custom icons or HTML. We would need to render it and then replace the innerHTML afterwards. But yes, I agree, an icon would be much better

Reply:
Can we use our own button, outside the calendar, next to the timezone dropdown?